Anlab.Core/Models/OrderDetails.cs

```diff
@@ -23,6 +23,11 @@ public class OrderDetails
 
         public string LabComments { get; set; }
         public decimal AdjustmentAmount { get; set; }
+        public decimal GrandTotal { get

```

Comment:
I think we added it so that we can show the previous price and the new adjusted one to the user